### PR TITLE
Fix Push-Location.md and Pop-Location.md

### PR DESCRIPTION
**Pull request by sethvs**
# PR Summary

- Make **StackName** parameter name bold
- Remove quotes around "stack" in Push-Location.md (as it is in Pop-Location.md)
- Break a long line.
- Remove duplicate `PowerShell lets you store provider locations in location stacks.` sentence.
- Remove boldness from **Location** in Pop-Location.md (as it is in Push-Location.md)
- Add links to **about_Aliases.md** and **about_Providers.md**

## PR Context

Select the area of the Table of Contents containing the documents being changed.

**Conceptual content**
- [ ] Overview and Install
- [ ] Learning PowerShell
  - [ ] PowerShell 101
  - [ ] Deep dives
  - [ ] Remoting
- [ ] Release notes (What's New)
- [ ] Windows PowerShell
  - WMF, ISE, release notes, etc.
- [ ] DSC articles
- [ ] Community resources
- [ ] Sample scripts
- [ ] Gallery articles
- [ ] Scripting and development
  - [ ] Legacy SDK

**Cmdlet reference & about_ topics**
- [x] Version 7.1 preview content
- [x] Version 7.0 content
- [x] Version 6 content
- [x] Version 5.1 content

## PR Checklist

- [x] I have read the [contributors guide][contrib] and followed the style and process guidelines
- [x] PR has a meaningful title
- [x] PR is targeted at the _staging_ branch
- [x] All relevant versions updated
- [ ] Includes content related to issues and PRs - see [Closing issues using keywords][key].
- [x] This PR is ready to merge and is not **Work in Progress**
  - If the PR is work in progress, please add the prefix `WIP:` or `[WIP]` to the beginning of the
    title and remove the prefix when the PR is ready.

[contrib]: https://docs.microsoft.com/powershell/scripting/community/contributing/overview
[key]: https://help.github.com/en/articles/closing-issues-using-keywords

### CODE REVIEW

**Comment by sethvs** [comment]
@sdwheeler There are links to other .md files in Push-Location.md 
```
You can also refer to `Push-Location` by its built-in alias, `pushd`. For more information, see
[about_Aliases](../Microsoft.PowerShell.Core/About/about_Aliases.md).

The `Push-Location` cmdlet is designed to work with the data exposed by any provider. To list the
providers available in your session, type `Get-PSProvider`. For more information, see
[about_Providers](../Microsoft.PowerShell.Core/About/about_Providers.md).
```
and not in Pop-Location.md
```
You can also refer to `Pop-Location` by its built-in alias, `popd`. For more information, see
about_Aliases.

`Pop-Location` is designed to work with the data exposed by any provider. To list the providers
available in your session, type `Get-PSProvider`. For more information, see about_Providers.
```
Should we add the links?

The question is because these links are useful in online help, but useless (though still present) in PowerShell local help.

File: reference/5.1/Microsoft.PowerShell.Management/Push-Location.md [hunk 1]
```diff
@@ -252,7 +252,7 @@ The unnamed default location stack is fully accessible only when it is the curre
 If you make a named location stack the current location stack, you can no longer use
 `Push-Location` or `Pop-Location` cmdlets to add or get items from the default stack or use
 `Get-Location` command to display the locations in the unnamed stack. To make the unnamed stack the
-current stack, use the StackName parameter of the `Set-Location` cmdlet with a value of $null or an
+current stack, use the **StackName** parameter of the `Set-Location` cmdlet with a value of $null or an
```
Comment (sdwheeler) [inline reference/5.1/Microsoft.PowerShell.Management/Push-Location.md:255]:
```suggestion
current stack, use the **StackName** parameter of the `Set-Location` cmdlet with a value of `$null` or an
```

File: reference/6/Microsoft.PowerShell.Management/Push-Location.md [hunk 2]
```diff
@@ -234,7 +234,7 @@ The unnamed default location stack is fully accessible only when it is the curre
 If you make a named location stack the current location stack, you can no longer use
 `Push-Location` or `Pop-Location` cmdlets to add or get items from the default stack or use
 `Get-Location` command to display the locations in the unnamed stack. To make the unnamed stack the
-current stack, use the StackName parameter of the `Set-Location` cmdlet with a value of $null or an
+current stack, use the **StackName** parameter of the `Set-Location` cmdlet with a value of $null or an
```
Comment (sdwheeler) [inline reference/6/Microsoft.PowerShell.Management/Push-Location.md:237]:
```suggestion
current stack, use the **StackName** parameter of the `Set-Location` cmdlet with a value of `$null` or an
```

File: reference/7.0/Microsoft.PowerShell.Management/Push-Location.md [hunk 3]
```diff
@@ -234,7 +234,7 @@ The unnamed default location stack is fully accessible only when it is the curre
 If you make a named location stack the current location stack, you can no longer use
 `Push-Location` or `Pop-Location` cmdlets to add or get items from the default stack or use
 `Get-Location` command to display the locations in the unnamed stack. To make the unnamed stack the
-current stack, use the StackName parameter of the `Set-Location` cmdlet with a value of $null or an
+current stack, use the **StackName** parameter of the `Set-Location` cmdlet with a value of $null or an
```
Comment (sdwheeler) [inline reference/7.0/Microsoft.PowerShell.Management/Push-Location.md:237]:
```suggestion
current stack, use the **StackName** parameter of the `Set-Location` cmdlet with a value of `$null` or an
```

File: reference/5.1/Microsoft.PowerShell.Management/Pop-Location.md [hunk 4]
```diff
@@ -163,13 +164,12 @@ A stack is a last-in, first-out list in which only the most recently added item
...
-specify a stack name, PowerShell uses the current location stack. By default, the unnamed
-default location is the current location stack, but you can use the `Set-Location` cmdlet to change
-the current location stack.
+PowerShell creates an unnamed default location stack and you can create multiple named location
+stacks. If you do not specify a stack name, PowerShell uses the current location stack. By
+default, the unnamed default location is the current location stack, but you can use the
+`Set-Location` cmdlet to change the current location stack.
 
-To manage location stacks, use the PowerShell**Location** cmdlets, as follows:
+To manage location stacks, use the PowerShell Location cmdlets, as follows:
```
Comment (sdwheeler) [inline reference/5.1/Microsoft.PowerShell.Management/Pop-Location.md:172]: Let's use this format throughout the articles:
```suggestion
To manage location stacks, use the PowerShell `*-Location` cmdlets, as follows:
```

File: reference/6/Microsoft.PowerShell.Management/Pop-Location.md [hunk 5]
```diff
@@ -145,13 +146,12 @@ A stack is a last-in, first-out list in which only the most recently added item
...
-specify a stack name, PowerShell uses the current location stack. By default, the unnamed
-default location is the current location stack, but you can use the `Set-Location` cmdlet to change
-the current location stack.
+PowerShell creates an unnamed default location stack and you can create multiple named location
+stacks. If you do not specify a stack name, PowerShell uses the current location stack. By
+default, the unnamed default location is the current location stack, but you can use the
+`Set-Location` cmdlet to change the current location stack.
 
-To manage location stacks, use the PowerShell**Location** cmdlets, as follows:
+To manage location stacks, use the PowerShell Location cmdlets, as follows:
```
Comment (sdwheeler) [inline reference/6/Microsoft.PowerShell.Management/Pop-Location.md:154]:
```suggestion
To manage location stacks, use the PowerShell `*-Location` cmdlets, as follows:
```

File: reference/7.0/Microsoft.PowerShell.Management/Pop-Location.md [hunk 6]
```diff
@@ -145,13 +146,12 @@ A stack is a last-in, first-out list in which only the most recently added item
...
-specify a stack name, PowerShell uses the current location stack. By default, the unnamed
-default location is the current location stack, but you can use the `Set-Location` cmdlet to change
-the current location stack.
+PowerShell creates an unnamed default location stack and you can create multiple named location
+stacks. If you do not specify a stack name, PowerShell uses the current location stack. By
+default, the unnamed default location is the current location stack, but you can use the
+`Set-Location` cmdlet to change the current location stack.
 
-To manage location stacks, use the PowerShell**Location** cmdlets, as follows:
+To manage location stacks, use the PowerShell Location cmdlets, as follows:
```
Comment (sdwheeler) [inline reference/7.0/Microsoft.PowerShell.Management/Pop-Location.md:154]:
```suggestion
To manage location stacks, use the PowerShell `*-Location` cmdlets, as follows:
```

File: reference/7.1/Microsoft.PowerShell.Management/Pop-Location.md [hunk 7]
```diff
@@ -145,13 +146,12 @@ A stack is a last-in, first-out list in which only the most recently added item
...
-specify a stack name, PowerShell uses the current location stack. By default, the unnamed
-default location is the current location stack, but you can use the `Set-Location` cmdlet to change
-the current location stack.
+PowerShell creates an unnamed default location stack and you can create multiple named location
+stacks. If you do not specify a stack name, PowerShell uses the current location stack. By
+default, the unnamed default location is the current location stack, but you can use the
+`Set-Location` cmdlet to change the current location stack.
 
-To manage location stacks, use the PowerShell**Location** cmdlets, as follows:
+To manage location stacks, use the PowerShell Location cmdlets, as follows:
```
Comment (sdwheeler) [inline reference/7.1/Microsoft.PowerShell.Management/Pop-Location.md:154]:
```suggestion
To manage location stacks, use the PowerShell `*-Location` cmdlets, as follows:
```

**Comment by sethvs** [comment]
Done.

Should we add **about_CommonParameters**, **about_Aliases**, and **about_Providers** to `Related Links` section since they are mentioned in the file?

**Pull request merged**